### DISCUSSION TITLE
Will this project convert JSON to C++?

### DISCUSSION THREAD

**Discussion by MusicNearMe** [nlohmann/json#2784] [Q&A] [answered]
Hello, I had a really quick question for you.

Can this project convert a JSON object to a C++ class? 

I do understand that it can parse and serialize JSON. 

If so, how would one use the conversion? 
If not, are there any other tools you might recommend?

Thanks

**Comment by nlohmann** [comment, marked as answer]
Yes, this library can [parse](https://json.nlohmann.me/api/basic_json/parse/) a JSON text into a C++ class `nlohmann::json` which you then can access like any other C++ container such as a `std::array` or a `std::map`. For instance, if you parse `{"foo" : 42}` to a variable `j`, you can access the value `42` with `j["foo"]`.

**Reply by nlohmann** [reply]
Do you need further assistance with this issue?

**Comment by MusicNearMe** [comment]
I do not, thanks for the responses and checking back. The plugin is working
quite well as I understand it. Thanks again
- Josef
